app/assets/stylesheets/provider/_buttons.scss

```diff
@@ -84,37 +73,34 @@ form.formtastic .button-bar, .button-bar {
 
   & > fieldset {
 
-    &.actions:first-child {
-      position: absolute;
-      left: 0;
+    &.actions:last-child {
```

Comment:
`:first-child` and `:last-child` are swapped, so that `first-child` has a preference. The result of this is that if there is only 1 set of actions (so, it is both the first and the last child), it is treated as the first child and the action buttons are aligned to the left of the form.